build/template-pack-and-sign-all-nugets.yaml

```diff
@@ -32,7 +32,7 @@ steps:
 - task: CopyFiles@2
   displayName: 'Copy Files to: $(Build.ArtifactStagingDirectory)\packages'
   inputs:
-    SourceFolder: '$(Build.SourcesDirectory)'
+    SourceFolder: '$(Build.ArtifactStagingDirectory)'
```

Comment:
With dotnet pack, the packages are not generated in the source directory, but in the ArtifactStagingDirectory